doc/design.rst

```diff
@@ -350,6 +359,11 @@ Design for mlx.traceability
 
     An overview table of the attribute values for documentation parts shall be generated.
 
+.. item:: DESIGN-ATTRIBUTE_SORT Custom sorting of items' attributes
+    :depends_on: DESIGN-ATTRIBUTES
+
+    The plugin shall have a directive that allows configurability of the order of items' attributes.
```

Comment:
Shall=has as these are not requirements so describe what is done.

Reply:
Maybe alao add how sorting and filtering are working?

Reply:
Nevermind last comment= i hace seen the details in usage

Reply:
Fixed design items in 8f2f510a77736abca463ea379c69ee2d5bd5238e